packages/design-system/src/components/button.stories.tsx

```diff
@@ -1,10 +1,94 @@
-import type { ComponentStory } from "@storybook/react";
-import { Button } from "./button";
+import { type ComponentProps } from "react";
+import { MenuIcon, CrossIcon, TrashIcon } from "@webstudio-is/icons";
+import { Button as ButtonComponent } from "./button";
 
 export default {
-  component: Button,
+  title: "Library/Button",
 };
 
-export const Simple: ComponentStory<typeof Button> = () => {
-  return <Button>Simple</Button>;
+const iconsMap = {
+  undefined: undefined,
+  "<MenuIcon>": <MenuIcon />,
+  "<CrossIcon>": <CrossIcon />,
+  "<TrashIcon>": <TrashIcon />,
+} as const;
+
+const Section = ({
+  title,
+  children,
+}: {
+  title: string;
+  children: React.ReactNode;
+}) => (
+  <>
+    <h3 style={{ fontFamily: "sans-serif" }}>{title}</h3>
+    <div style={{ display: "flex", gap: 12 }}>{children}</div>
+  </>
+);
+
+export const Button = ({
+  icon,
+  ...rest
+}: Omit<ComponentProps<typeof ButtonComponent>, "icon"> & {
+  icon?: keyof typeof iconsMap;
+}) => (
+  <>
+    <Section title="Configurable">
+      <ButtonComponent icon={icon && iconsMap[icon]} {...rest} />
+    </Section>
+
+    <Section title="Variants">
+      {variants.map((variant) => (
+        <ButtonComponent variant={variant} key={variant}>
+          {variant}
+        </ButtonComponent>
+      ))}
+    </Section>
+
+    <Section title="With Icon">
+      <ButtonComponent icon={<TrashIcon />}>Button</ButtonComponent>
+      <ButtonComponent icon={<TrashIcon />} iconPosition="right">
```

Comment:
do we need to call it icon? is there anything icon specific? or could it be just prefix/suffix?

Reply:
Nothing icon specific, could be prefix/suffix. 

Not completely sure it's a good idea. It encourages to use component in a way not intended by our design in Figma, which I'm also not sure whether it's a good or bad thing...

Will switch to `prefix/suffix` as I'm not sure either way and it's a bit more straightforward API than `icon + iconPosition`